mozilla-release/toolkit/components/telemetry/TelemetryEnvironment.jsm

```diff
@@ -108,7 +108,8 @@ const DEFAULT_ENVIRONMENT_PREFS = new Map([
   ["browser.shell.checkDefaultBrowser", {what: RECORD_PREF_VALUE}],
   ["browser.search.suggest.enabled", {what: RECORD_PREF_VALUE}],
   ["browser.startup.homepage", {what: RECORD_PREF_STATE}],
-  ["browser.startup.page", {what: RECORD_PREF_VALUE}],
+  ["browser.startup.restoreTabs", {what: RECORD_PREF_VALUE}],
+  ["browser.startup.addFreshTab", {what: RECORD_PREF_VALUE}],
```

Comment:
@maxim-cliqz building on the previous comment - using a different value will allow us to keep this file as it was with no changes